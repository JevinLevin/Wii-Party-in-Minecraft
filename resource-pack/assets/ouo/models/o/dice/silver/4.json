{
	"credit": "OuO Studio Creation",
	"parent": "ouo:o/dice/parent/4",
	"textures": {
		"1": "ouo:o/dice/silver",
		"particle": "ouo:o/dice/silver"
	}
}