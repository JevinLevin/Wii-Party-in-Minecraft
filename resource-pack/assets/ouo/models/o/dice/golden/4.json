{
	"credit": "OuO Studio Creation",
	"parent": "ouo:o/dice/parent/4",
	"textures": {
		"1": "ouo:o/dice/golden",
		"particle": "ouo:o/dice/golden"
	}
}